### PR TITLE
changelog: Update changelog for 5.0.0 release

### PR DESCRIPTION
**Pull request by jmt-lab**
**Description of changes:**
* Updates changelog for 5.0.0 release that expects to be used with the kernel kit

**Terms of contribution:**

By submitting this pull request, I agree that this contribution is dual-licensed under the terms of both the Apache License, version 2.0, and the MIT license.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,26 @@
+# v5.0.0 (2024-12-12)
+
+## OS Changes
+* Update binutil to 2.41 ([#306])
+
+## Build Changes
+* Update bottlerocket-sdk to v0.5.0 ([#306])
+* Remove the following packages that move to the kernel-kit (#[313])
```
Comment (vigh-m) [inline CHANGELOG.md]:
```suggestion
* Remove the following packages that move to the kernel-kit ([#313])
```

Comment (jmt-lab) [reply CHANGELOG.md]: oops

**Review by rpkelly — APPROVED** [review]

**Review by vigh-m — APPROVED** [review]

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,26 @@
+# v5.0.0 (2024-12-12)
+
+## OS Changes
+* Update binutil to 2.41 ([#306])
```
Comment (ginglis13) [inline CHANGELOG.md:4]:
```suggestion
* Update binutils to 2.41 ([#306])
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,26 @@
...
+
+## Build Changes
+* Update bottlerocket-sdk to v0.5.0 ([#306])
+* Remove the following packages that move to the kernel-kit ([#313])
```
Comment (ginglis13) [inline CHANGELOG.md:8]:
```suggestion
* Remove the following packages and migrate them to the [kernel-kit](https://github.com/bottlerocket-os/bottlerocket-kernel-kit) ([#313])

**Pull request merged**